### PR TITLE
Add diagram db file

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello dear reviewer!
Kindly find in this PR the lessons learned about Database Diagram built using [DrawSql](https://drawsql.app/teams/class-10/diagrams/vet-clinic).

We both created a diagram from our vet clinic database and converted this [Diagram](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/images/clinic_diagram.png) to a database script located in `schema_based_on_diagram.sql` file as required for this activity.

let us know if anything needs improvement or if we can merge this PR and move on to the next phase of the project.
Best regards 🎊 🤝 

image

### CODE REVIEW

**Review by andersonlebon — COMMENTED** [review]
## Hi @chiztechnology 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
image

You have submitted a project, that contains empty sections in the readme. kindly make sure the readme file is professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @chiztechnology, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
hello dear @Mihndim2020
I can't see any comment from you! could you please share so they can help improve my project please ?

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+);
+
+CREATE TABLE invoice_items(
+    id INT SERIAL PRIMARY KEY,
+    unit_price DECIMAL,
+    quantity int,
+    total_price DECIMAL,
+    invoice_id INT FOREIGN KEY references invoices(id),
+    treatment_id INT FOREIGN KEY references treatments(id)
+);
```
Comment (Mihndim2020) [inline schema_based_on_diagram.sql]: - [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md)for this project,  "`Join tables from many-to-many relationships might not appear in the diagram, but you still need them.`", a "join table" for the many-to-many relastionships should be created, even if they are not showed on the diagram. Therefore you should create a table between medical_histories and treatments, something like so:   

```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

- [ ] As it s a [requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md)for this project, "`Remember to add the FK indexes.`", please make sure all FOREIGN KEYS have indexes created, something like so: 

* `CREATE INDEX ON medical_histories (patient_id);`
* `CREATE INDEX ON invoices (medical_history_id);`
* `CREATE INDEX ON invoice_items (invoice_id);`
* `CREATE INDEX ON invoice_items (treatment_id);`
* `CREATE INDEX ON medical_histories_has_treatments (medical_history_id);`
* `CREATE INDEX ON medical_histories_has_treatments (treatment_id);`

##### Please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @chiztechnology, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (chiztechnology) [reply schema_based_on_diagram.sql]: owh. that's amazing. thank you for your great review! let me check it out and apply this to improve my code.
Thanks alot.

**Review by BertrandConxy — APPROVED** [review]
Hi @chiztechnology, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**